test/api.spec.js

```diff
@@ -0,0 +1,1397 @@
+import { api } from '../src';
+
+describe('Api', function() {
+  describe('getList', function() {
+    beforeEach(function() {
+      this.server = sinon.createFakeServer();
+    });
+
+    afterEach(function() {
+      this.server.restore();
+    });
+
+    it('should return the requested list', async function() {
+      const options = {
+        userId: 'user123456',
+        apiKey: 'sample',
+        secretKey: 'drH5zDpDMLLA6elOPVl7tQ==',
```

Comment:
essa secret key é de vdd? se for, não acho legal comitar, na vdd se n for pública já deveriamos mudá-la

Reply:
já mudei